plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3630,6 +3634,38 @@ public void testEmptyDelete()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCtasFromAVersionedTable()
+            throws Exception
+    {
+        String sourceTableName = "test_ctas_versioned_source_" + randomTableSuffix();
+        String sinkTableNameVersion = "test_ctas_versioned_sink_" + randomTableSuffix();
+        String sinkTableNameTimestamp = "test_ctas_versioned_sink_" + randomTableSuffix();
+
+        assertUpdate("CREATE TABLE " + sourceTableName + "(an_integer integer)");
+        Thread.sleep(1);
+        assertUpdate("INSERT INTO " + sourceTableName + " VALUES (1), (2), (3)", 3);
```

Comment:
Same comment elsewhere
```suggestion
        assertUpdate("INSERT INTO " + sourceTableName + " VALUES 1, 2, 3", 3);
```